### ISSUE TITLE
First card blank after looping

### ISSUE THREAD

**Issue by Millennium-bug-82** [bramkragten/swipe-card#69]
I really love this card but the loop drives me crazy, I can't get it to work properly no matter what I try.

This is what I use but the details don't really matter much right now. 
The problem is that the card swipes and loops just fine except for one thing. When I swipe from the last card to the "next" card it returns to card 1 as it's supposed to do, the first card shows as blank however. When I start dragging the card it appears as normal. This leads me to believe the card is actually there, just not rendering correctly. Clearing browser cache doesn't fix anything. I've tried Chrome, Edge, Firefox and Fully Kiosk browser and all give me the same result. Is this a bug or am I doing something wrong.

`type: custom:swipe-card
start_card: 1
reset_after: 30
parameters:
  effect: coverflow
  spaceBetween: 8
  grabCursor: true
  centeredSlides: true
  pagination:
    type: progressbar
  loop: true
  slidesPerView: 1
`

**Comment by Madelena** [comment]
Yeah, looping a swipe-card does this to me too. The only way for me to fix it is to not make it loop.

**Comment by Polarisbear** [comment]
Hey, same issue here, let me know if you need additional information
This is the current card config:

```
type: custom:swipe-card
parameters:
  loop: true
  effect: slide
  followFinger: false
```

**Comment by seanpdiaz** [comment]
Just wanted to say that I have the exact same issue here, no matter what options I use from SwiperJS I can't get it to loop. Would be curious to know if anyone has gotten this to work or not. Additionally curious if @bramkragten is still actively developing this or if it is deprecated?

**Comment by teek541** [comment]
same issue here.

**Comment by IanAdd** [comment]
I have the same problem - loop gives blank scrolling left or right beyond last one.
Also I can't seem to get startcard: 2 to work either.

**Comment by captaincanada** [comment]
same here. 
loop= true is not working correctly...
would be great to have a workaround 

thx to all, working on this issue

**Comment by chrannen** [comment]
Same Problem here. Paramters "loop: true" does not work.

**Comment by ColtonYYZ** [comment]
Following as I have the same problem
